File: com.vorwardit.wordsworthcms/Models/Layout.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace com.vorwardit.wordsworthcms.Models
{
    public class Layout
    {
        public Guid LayoutId { get; set; }

        public Guid SiteId { get; set; }
        [ForeignKey("SiteId")]
        public Site Site { get; set; }

        [MaxLength(50)]
        public string Name { get; set; }

        public string Body { get; set; }
    }
}

File: com.vorwardit.wordsworthcms/lib/admin/Domain/Layout.ts
module app.domain {
    export interface ILayout {
        layoutId: string;
        siteId: string;
        name: string;
        body: string;
    }

    export class Layout implements ILayout {
        constructor(
            public layoutId: string,
            public siteId: string,
            public name: string,
            public body: string
        ) {
        }
    }
}